src/AnnotationDialog.js

```diff
@@ -614,7 +614,27 @@ class AnnotationDialog extends EventEmitter {
             hour: '2-digit',
             minute: '2-digit'
         });
-        const text = util.htmlEscape(annotation.text);
+        const annotationText = annotation.text;
```

Comment:
Can we put a refactored version of this in a util method?

Reply:
DONE